bower.json

```diff
@@ -28,13 +28,13 @@
   ],
   "dependencies": {
     "polymer": "^2.0.0",
-    "vaadin-themable-mixin": "vaadin/vaadin-themable-mixin#^1.4.1",
-    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.0.0",
-    "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.3.0",
-    "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.2.0",
+    "vaadin-themable-mixin": "vaadin/vaadin-themable-mixin#^1.5.2",
+    "vaadin-element-mixin": "vaadin/vaadin-element-mixin#^2.3.1",
+    "vaadin-lumo-styles": "vaadin/vaadin-lumo-styles#^1.6.0",
+    "vaadin-material-styles": "vaadin/vaadin-material-styles#^1.3.2",
     "vaadin-text-field": "vaadin/vaadin-text-field#^2.3.0",
```

Comment:
There is a `text-field` with `rtl` styles: https://github.com/vaadin/vaadin-text-field/releases/tag/v2.6.0-alpha3

Reply:
Done.